### ISSUE TITLE
WSL2 fails to start with networkingMode=bridged

### ISSUE THREAD

**Issue by Kilill** [microsoft/WSL#9218]
### Version

Microsoft Windows [Version 10.0.19045.2311]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

Linux version 5.15.74.2-microsoft-standard-WSL2 (oe-user@oe-host) (x86_64-msft-linux-gcc (GCC) 9.3.0, GNU ld (GNU Binutils) 2.34.0.20200220) #1 SMP Wed Nov 2 19:50:29 UTC 2022

### Distro Version

 Ubuntu 20.04.3 LTS

### Repro Steps

create a virtual switch according to : [(https://github.com/luxzg/WSL2-fixes/blob/master/networkingMode%3Dbridged.md#install-wsl-2-preview)]

fresh install of wsl:
`wsl --install`
terminated and shutdown wsl

add the lines:
```
[wsl2]
networkingMode=bridged
vmSwitch=WSL_external
```
to `.wslconfig`

try runnig wsl

### Expected Behavior

wsl should start

### Actual Behavior

wsl command terminates with:

```
The virtual machine or container JSON document is invalid.
Error code: Wsl/Service/CreateInstance/CreateVm/ConfigureNetworking/0x8037010d
```

removing the `networkingMode=bridged` line from the config allows wsl to start

**Comment by elsaco** [comment]
@Kilill WSL network bridging, as per mentioned guide, works in Windows 11 only. Looks like you're using Windows 10.

**Comment by Kilill** [comment]
Yeah noticed that, but afaik that guide was also done for the prerelease of WSL 2 before it transitioned into V1.0 and supposedly supported on W10...

**Comment by elsaco** [comment]
I have it working on Windows 11 but on Windows 10 it keeps throwing errors:

```
The virtual machine or container JSON document is invalid.
Error code: Wsl/Service/CreateInstance/CreateVm/ConfigureNetworking/0x8037010d
The virtual machine or container JSON document is invalid.
Error code: Wsl/Service/CreateInstance/CreateVm/ConfigureNetworking/0x8037010d
The virtual machine or container JSON document is invalid.
Error code: Wsl/Service/CreateInstance/CreateVm/ConfigureNetworking/0x8037010d
Press any key to continue...
```

WSL version used for testing:

```
PS C:\Users\elsaco> wsl --version
WSL version: 1.0.0.0
Kernel version: 5.15.74.2
WSLg version: 1.0.47
MSRDC version: 1.2.3575
Direct3D version: 1.606.4
DXCore version: 10.0.25131.1002-220531-1700.rs-onecore-base2-hyp
Windows version: 10.0.19045.2311
```
